### PR TITLE
[AS] ADY-9 BDD scenarios, extra logging for Payout Service

### PR DESCRIPTION
**Pull request by asamrae2y**
…rence upon successful post

### CODE REVIEW

File: src/test/features/adyennotifications/adyenaccountholderpayout.feature [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Feature: Payout notifications for seller payout
+
+    @ADY-9
+    Scenario: Successful payout notification is received upon successful seller payout
+        Given a AccountHolder exists who has passed KYC checks and is eligible for payout
```
Comment (rmcotton) [inline src/test/features/adyennotifications/adyenaccountholderpayout.feature]: KYC checks is a detail we dont need. Should just be an AccountHolder exists that is eligible for payout

Its part of the KYC testing that we check that they switch from ineligible to eligible for payout.

File: src/test/features/adyennotifications/adyenaccountholderpayout.feature [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            | seller       |
+            | PayoutShop01 |
+        When a payment voucher is sent to the App from Mirakl
```
Comment (rmcotton) [inline src/test/features/adyennotifications/adyenaccountholderpayout.feature]: Use Connector rather than App

Comment (rmcotton) [reply src/test/features/adyennotifications/adyenaccountholderpayout.feature]: dont use Mirakl if it actually isnt being sent from Mirakl as its misleading. When its really sent from Mirakl, we'lll say in the scenario.

File: src/test/features/adyennotifications/adyenaccountholderpayout.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        Then adyen will send the ACCOUNT_HOLDER_PAYOUT notification
+            | currency | amount | statusCode | iban                   |
+            | EUR      | 29.14  | Initiated  | GB26TEST40051512347366 |
+
+    @ADY-9
+    Scenario: Failure status is received for payout notification
+        Given a AccountHolder exists who has passed KYC checks and is eligible for payout
```
Comment (rmcotton) [inline src/test/features/adyennotifications/adyenaccountholderpayout.feature]: isnt it -->        Given a AccountHolder exists who is NOT eligible for payout

File: src/test/resources/config/application.yml [hunk 4]
```diff
@@ -113,6 +113,8 @@ shops:
     shopIds:
         Cottons Shop: 2000
         Samras Supermarket: 2001
+        PayoutShop01: 4222
```
Comment (rmcotton) [inline src/test/resources/config/application.yml]: good idea.

File: src/test/features/adyennotifications/adyenaccountholderpayout.feature (hunk 3 above)
Comment (asamrae2y) [reply src/test/features/adyennotifications/adyenaccountholderpayout.feature]: yup you're right. originally my idea of the unhappy path was going to be that the accountHolder is eligable and we send the payment voucher to it. But I think you're right, non-eligable should not be receiving payments.

**Comment by rmcotton** [comment]
@asamrae2y you can have a crack at fixing the 1 sonar issue if you like (in src not testSrc). Otherwise approved and merge when you like. Ticket obviously cannot be done until we get the MockMvc working.

**Pull request merged**